packages/cardano-services-client/src/AssetInfoProvider/assetInfoHttpProvider.ts

```diff
@@ -10,9 +12,19 @@ export const defaultAssetProviderPaths: HttpProviderConfigPaths<AssetProvider> =
  * Connect to a Cardano Services HttpServer instance with the service available
  *
  * @param {string} baseUrl server root url, w/o trailing /
+ * @param paths A mapping between provider method names and url paths. Paths have to use /leadingSlash
+ * @param adapter This adapter that allows to you to modify the way Axios make requests.
+ * @param logger The logger instance.
  */
-export const assetInfoHttpProvider = (baseUrl: string, paths = defaultAssetProviderPaths): AssetProvider =>
+export const assetInfoHttpProvider = (
+  baseUrl: string,
+  paths = defaultAssetProviderPaths,
+  adapter?: AxiosAdapter,
+  logger = dummyLogger
```

Comment:
Is the scope of ADP-1933 only server side packages?
Otherwise `dummyLogger` (or dummy stuff in general) should be used only in tests.
The same comment apply to other files as well.

Reply:
> Is the scope of ADP-1933 only server side packages?

No. Good catch @iccicci

Reply:
Right, so logger should be a required param

Reply:
addressed here https://github.com/input-output-hk/cardano-js-sdk/commit/78c257176933121ad83152ddb17d1e555b0f8b35 , logger is now a required parameter, also, in order to remove positional parameters in the services factory methods, I created a new config object.